lib/internal/modules/esm/module_job.js

```diff
@@ -101,8 +101,7 @@ class ModuleJob {
 
   async run() {
     const module = await this.instantiate();
-    module.evaluate(-1, false);
-    return module;
+    return { module, result: module.evaluate(-1, false) };
```

Comment:
it might be worth documenting what `-1 and `false` indicate, it's a little magical ... I believe it's?

`module.evaluate(timeout, breakOnSigint)`?

Reply:
/cc @guybedford on this one too 😅

Reply:
This is also on master and defined with its function signature at https://github.com/nodejs/node/blob/master/src/module_wrap.cc#L274.

Reply:
not a blocker, took me a while to figure out what the heck `-1` and `false` represented, so thought it might be worth documenting in a variety of places.

Reply:
fixed in 43238fa66d